README.md

```diff
@@ -298,11 +289,30 @@ if err != nil {
 fmt.Print(status, status.ChannelId)
 ```
 
+## Resources
+
```

Comment:
The Resources section of the template (https://github.com/ably/ably-common/blob/main/templates/sdk-readme.md#resources) says this section should have an index of ably-labs demo projects which use the ably-go SDK.

I did a quick filter on ably-labs to see the Go projects there (https://github.com/ably-labs?q=&type=all&language=go&sort=) 

I think we can add something like this which lists all those projects in alphabetical order:
```
Demo repositories hosted at [ably-labs](https://github.com/ably-labs) which use `ably-go`.  

* [Ableye](https://github.com/ably-labs/Ableye)
* [AblyD](https://github.com/ably-labs/AblyD)
* [Ably Go Terminal Chat](https://github.com/ably-labs/go-chat)
* [Broadcasting messages in Go with Ably and Redis](https://github.com/ably-labs/broadcasting-redis-go)
* [Sync Edit](https://github.com/ably-labs/sync-edit)
* [Word Game](https://github.com/ably-labs/word-game)
```